resources/CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ set(RESOURCES_FILES
   Shaders/TexturedVertexShader.glsl
   Shaders/frag.spv
   Shaders/vert.spv
+
+  Sounds/waltz.ogg
+  Sounds/lazer.ogg
```

Comment:
UK spelling please (or at least the meme spelling :D )

Reply:
Yep, I messed up memes - will fix, very disappoint

Reply:
Done